cmd/app/command/device-list.go

```diff
@@ -18,11 +18,28 @@ package command
 
 import (
 	goflag "flag"
+	"fmt"
+	"html/template"
+	"os"
 
 	"github.com/openebs/node-disk-manager/cmd/controller"
 	"github.com/spf13/cobra"
 )
 
+// defaultDeviceList template use to print list of device
+const defaultDeviceList = `
```

Comment:
Good. Can you provide the details on how the template will render in code comments. I know you have done that in PR comment. But this is the place where we should.

Reply:
done.